scripts/steam.js

```diff
@@ -1,34 +1,25 @@
-/* global require __dirname */
-const fs = require('fs')
+/* global require  module */
 const delay = require('delay')
-const {resolve} = require('path')
-const {steam, getTopRatedGames} = require('./lib')
+const {steam, load, saveToJson} = require('./lib')
 
-async function main() {
-  let games = []
-
-  console.log('Get top rated games from steamdb...')
-  try {
-    games = await getTopRatedGames()
-  } catch (e) {
-    console.log('Retrieval of top rated games from steamdb.info failed', e)
-  }
-
-  const gamesWithDetails = []
+const steamGames = load('steam-games')
```

Comment:
maybe reame the file to steam-app-details? / steam-game-details to distinguish it more from the top rated steam db games or our finale games.json respectively?